frontend/countries.html

```diff
@@ -204,7 +222,7 @@ <h1 class="page_title">Explore countries</h1>
                     </thead>
                     <tbody ng-if="!bLoadingTabel">
                     <!--  Country with dataset submitted -->
-                    <tr ng-repeat="country in allCountries | orderBy: sortField:sortDirection | filter: filterBy" ng-click="changepage('dataset_list.html?idcountry=' + country.country + '&idcategory=0')"
+                    <tr ng-repeat="country in allCountries | orderBy: sortField:sortDirection | filter: filterBy | filter: {name:countrySearchFilter, region:regionSearchFilter}" ng-click="changepage('dataset_list.html?idcountry=' + country.country + '&idcategory=0')"
```

Comment:
@CIMAManuel nice bit 👍 it makes the filter way easier. Thanks!

Reply:
@oncletom I have a doubt: every country can has more one values in region field. 
This filter, if i understand, works with only a value. 
@nastasi-oq

Reply:
If you return an array for the `region` or a new `group` field with an array of values, it will be easy to change on our side of things.

Reply:
I understand you are still working on this one, let me know when review is needed.
Thanks